cl/context.go

```diff
@@ -149,11 +146,10 @@ func CLCreateContextFromType(properties []CL_context_properties,
 		c_errcode_ret = CL_INVALID_VALUE
 		c_context = nil
 	} else {
-		var c_properties []C.cl_context_properties
 		var c_properties_ptr *C.cl_context_properties
 
 		if properties != nil {
-			c_properties = make([]C.cl_context_properties, len(properties))
+			c_properties := make([]C.cl_context_properties, len(properties))
```

Comment:
Looks like the `else` block is pointless with `c_properties_ptr = nil` changing nothing.